system/api-server/test/app.e2e-spec.ts

```diff
@@ -19,6 +19,6 @@ describe('AppController (e2e)', () => {
     return request(app.getHttpServer())
       .get('/')
       .expect(200)
-      .expect('Hello World!');
+      .expect('Hello World!'); 
```

Comment:
Where are the rest of the tests..?

Reply:
I initially thought there was a commit issue but after checking, I remembered I took them out because I had problems writing the tests for the create receipt endpoint because the upload part kept failing